File: mot-api/module/DvsaEntities/src/DvsaEntities/Entity/Colour.php
<?php
namespace DvsaEntities\Entity;

use Doctrine\ORM\Mapping as ORM;
use DvsaEntities\Entity\Person;
use DvsaEntities\EntityTrait\CommonIdentityTrait;

/**
 * Colour
 *
 * @ORM\Table(
 *  name="colour_lookup",
 *  indexes={
 *      @ORM\Index(name="fk_colour_created_by", columns={"created_by"}),
 *      @ORM\Index(name="fk_colour_last_updated_by", columns={"last_updated_by"})
 * })
 * @ORM\Entity(repositoryClass="DvsaEntities\Repository\ColourRepository", readOnly=true)
 */
class Colour extends Entity
{
    use CommonIdentityTrait;

    /**
     * @var string
     *
     * @ORM\Column(name="code", type="string", length=1, nullable=false)
     */
    private $code;

    /**
     * @var string
     *
     * @ORM\Column(name="name", type="string", length=45, nullable=false)
     */
    private $name;

    /**
     * @param string $code
     *
     * @return $this
     */
    public function setCode($code)
    {
        $this->code = $code;
        return $this;
    }

    /**
     * @return string
     */
    public function getCode()
    {
        return $this->code;
    }

    /**
     * @param string $name
     *
     * @return $this
     */
    public function setName($name)
    {
        $this->name = $name;
        return $this;
    }

    /**
     * @return string
     */
    public function getName()
    {
        return $this->name;
    }
}

File: mot-selenium/src/main/java/uk/gov/dvsa/domain/api/response/Colour.java
package uk.gov.dvsa.domain.api.response;


public class Colour {

    private String code;

    private String name;

    public String getCode() {
        return code;
    }

    public Colour setCode(String code) {
        this.code = code;
        return this;
    }

    public String getName() {
        return name;
    }

    public Colour setName(String name) {
        this.name = name;
        return this;
    }

    @Override
    public String toString() {
        return "Colour{" +
            ", code='" + code + '\'' +
            ", name='" + name + '\'' +
            '}';
    }
}
